src/crowbar.mli

```diff
@@ -58,12 +124,51 @@ val pp_bool : bool printer
 val pp_string : string printer
 val pp_list : 'a printer -> 'a list printer
 
+(** {1:top Testing} *)
+
+val add_test :
+  ?name:string -> ('f, unit) gens -> 'f -> unit
+(** [add_test name generators test_fn] adds [test_fn] to the list of eligible
+    tests to be run when the program is invoked.  At runtime, random data will
+    be sent to [generators] to create the input necessary to run [test_fn].  Any
+    failures will be printed annotated with [name]. *)
+
+(** {2 Aborting Tests} *)
 
+val guard : bool -> unit
+(** [guard b] will abort a test if [b] is false.  The test will not be recorded
```

Comment:
I usually prefer to describe effects of evaluation with the present tense. `bla` aborts ... The test is not recorded...